operations/alloy-mixin/config.libsonnet

```diff
@@ -3,10 +3,11 @@
         enableK8sCluster: true,
         enableAlloyCluster: true,
         enableLokiLogs: true,
-        filterSelector: 'job=~"$job"',
-        groupSelector: if self.enableK8sCluster then self.k8sClusterSelector + ', ' + self.filterSelector else self.filterSelector,
-        instanceSelector: self.groupSelector + ', instance=~"$instance"',
+        filterSelector: 'job=~"integrations/self"', #default job name used by alloy "self" exporter
         k8sClusterSelector: 'cluster=~"$cluster", namespace=~"$namespace"',
+        groupSelector: if self.enableK8sCluster then self.k8sClusterSelector + ', job="$job"' else 'job="$job"',        
+        instanceSelector: self.groupSelector + ', instance=~"$instance"',        
+        logsFilterSelector: 'service_name="alloy"', #use to filter logs originated from alloy, and avoid picking up other platform logs
```

Comment:
Yeah, I agree that ideally the same label set used to filter metrics should be present for logs, and we enforce that in integrations instructions and snippets. At the mixin level it is hard to enforce that, since there is no standard set of labels generated for logs.

The idea of this field is to give the customer the ability to configure it based on his own choice, and it may or may not match the topological set of labels that are present for metrics and generated by alloy. So that is why it is a separate field.

What is important is we provide the user means to filter alloy specific logs.

Reply:
That goes to say, I can make it empty by default, but will leave a comment that this should be set to avoid logs overlap.